Documentation/ApiOverview/Internationalization/TranslationServer/Crowdin/OnlineTranslation.rst

```diff
@@ -0,0 +1,120 @@
+
+4. Look into other localized projects in your language. There are tons of open source projects being translated, also into your language. Be curious and look at how the localization is done – there might be things to learn and adapt.
+
+5. Be consistent. Localization of high quality is characterised by the consistency. Make extensive use of the terms and glossary.
+
+6. Make careful use of machine translation. It’s tempting but dangerous to do a quick translation with one of the common machine translation tools and it can sometimes help you getting a deeper understanding of the meaning of a text.
+But very often a machine translated text breaks all of the rules above, unless you carefully rework the text afterwards.
+
+7. Work together. As in all other aspects of Open Source, things get so much better, if we work together. So, reach out for help if you get stuck. Or offer your knowledge if someone ask.
+Crowdin offers a good platform for cooperating with your team translators, and please join the `Translation Slack channel <https://typo3.slack.com/archives/C032FRT0W>`__.
+
+Translation Styles
+==================
+
+In general and where it makes sense, we follow the style guide from the Documentation Team: `<https://typo3.org/community/teams/content/writing-style-guide/>`__
+
+In the future when translation teams are getting bigger, it will be a good idea to develop local style guides.
+
+Links
+=====
+
+There are many good resources when it comes to translation, language, dictionaries, etc. Feel free to suggest your favorite websites, when you work with language.
+
+`<https://iate.europa.eu/home>`__
+
+`<https://gengo.com/translators/resources/>`__
+
+`<https://docs.microsoft.com/style-guide>`__
+
+FAQ
+===
+
+Should I localize both 9.5 and Master?
+--------------------------------------
+Master (which is currently version 10) is the leading version. Every string which exists in 9.5 as well is automatically filled during the export and only needs to be localized if it is different in 9.5.
+
+Strings are translated, but when are they taken into account and available for download?
+----------------------------------------------------------------------------------------
+As soon as a string is proofread, it will be taken into account at the next export. The export runs 3 times a day.
+
+How can I be sure what way a word, term or sting is to be translated?
+---------------------------------------------------------------------
+There are several ways to get help: In left panel you can either search the translation memory (TM) or the term base. You can also drop a comment to start a discussion or ask for advice.
+
+Where do I meet all the other translators?
+------------------------------------------
+There is a good chance to find help and endorsement in the TYPO3 Slack. Try the `Translation Slack channel <https://typo3.slack.com/archives/C032FRT0W>`__.
```

Comment:
Might want to add a general link to Slack with contains help for signing up.

Might want to add the name of the channel (in case the link does not work).